### PR TITLE
add TimeSpanObjectTypeVisitor

### PR DESCRIPTION
**Pull request by sinantutan**
As reported in #401 the type of TimeSpan is incorrectly displayed as a complex object. This PR aims for fixing this issue. 

We can then have a new property in our Order class of the type TimeSpan
image

And achieve the following result
image

### CODE REVIEW

**Comment by sinantutan** [comment]
@microsoft-github-policy-service agree

File: src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Visitors/TimeSpanObjectTypeVisitor.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        public TimeSpanObjectTypeVisitor(VisitorCollection visitorCollection) : base(visitorCollection)
+        {
+        }
+
+        /// <inheritdoc />
+        public override OpenApiSchema ParameterVisit(Type type, NamingStrategy namingStrategy)
+        {
+            return this.ParameterVisit(dataType: "string", dataFormat: "timespan");
+        }
+
```
Comment (justinyoo) [inline src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Visitors/TimeSpanObjectTypeVisitor.cs:21]: This requires `IsParameterVisitable(Type type)` method to be implemented.

**Review by justinyoo — CHANGES_REQUESTED** [review]
@sinantutan Thanks for your PR! I left a comment. In addition to that, would you be able to add tests for your changes, please?

**Comment by sinantutan** [comment]
> @sinantutan Thanks for your PR! I left a comment. In addition to that, would you be able to add tests for your changes, please?

@justinyoo many thanks for reviewing my PR. I've made the changes you requested. Please let me know if anything is missing.

**Review by justinyoo — APPROVED** [review]
Thanks @sinantutan for your contribution! LGTM

**Pull request merged**